src/ert/_c_wrappers/job_queue/queue.py

```diff
@@ -118,19 +118,11 @@ def __repr__(self) -> str:
             self.num_pending,
         )
         isrun = "running" if self.isRunning else "not running"
-        return self._create_repr(
+        return self._create_repr(  # type: ignore
             f"{isrun}, num_running={nrun}, num_complete={ncom}, "
             f"num_waiting={nwait}, num_pending={npend}"
         )
 
-    def __str__(self) -> str:
```

Comment:
Assuming you now regard this function is superfluous? (maybe never triggered in tests?)

Reply:
Fixed